e2e/teams.spec.ts

```diff
@@ -161,22 +167,26 @@ test.describe('Teams', () => {
           }),
         ).toBeVisible()
       })
+
+      await testAuth.step('user deletes team', async () => {
+        await goToDashboard(userPage)
+        await userPage.getByLabel('Teams').click()
+        await userPage.getByRole('link', { name: TEAM_NAME }).click()
+        await userPage.getByRole('button', { name: 'Team Actions' }).click()
+        await userPage.getByRole('link', { name: 'Delete Team' }).click()
+        await userPage.getByRole('button', { name: 'Delete' }).click()
+        await expect(
+          userPage.getByText('Team successfully deleted'),
+        ).toBeVisible()
+        await expect(
+          userPage.getByRole('link', { name: TEAM_NAME }),
+        ).not.toBeVisible()
+        teamId = undefined
+      })
     },
   )
 
   testAuth.afterAll(async ({ userPage, validatedUserPage }) => {
-    // delete team
-    await goToDashboard(userPage)
-    await userPage.getByLabel('Teams').click()
-    await userPage.getByRole('link', { name: TEAM_NAME }).click()
-    await userPage.getByRole('button', { name: 'Team Actions' }).click()
-    await userPage.getByRole('link', { name: 'Delete Team' }).click()
-    await userPage.getByRole('button', { name: 'Delete' }).click()
-    await expect(userPage.getByText('Team successfully deleted')).toBeVisible()
-    await expect(
-      userPage.getByRole('link', { name: TEAM_NAME }),
-    ).not.toBeVisible()
-
```

Comment:
This `afterAll` intermittently times out -- delete the team via the UI within the test itself, rather than in the `afterAll`. However, if the test fails before the team is deleted, then delete the team via the API in the `afterAll`, so that the users can still be cleaned up.